packages/manager/src/features/TopMenu/CreateMenu/CreateMenu.tsx

```diff
@@ -39,7 +39,8 @@ export interface CreateMenuLink extends BaseNavLink {
 export const CreateMenu = () => {
   const [anchorEl, setAnchorEl] = React.useState<HTMLElement | null>(null);
   const open = Boolean(anchorEl);
-
+  const theme = useTheme();
+  const isSmallScreen = useMediaQuery(theme.breakpoints.down('md'));
```

Comment:
Optionally, to avoid the need for the `useTheme` hook:

```suggestion
  import type { Theme } from '@mui/material/styles';

  const isSmallScreen = useMediaQuery((theme: Theme) =>
    theme.breakpoints.down('md')
  );
```

Reply:
yes! fixed